HOK.Utilities/HOK.ColorBasedIssueFinder/App.cs

```diff
@@ -0,0 +1,24 @@
+#region Namespaces
+using System;
+using System.Collections.Generic;
+using Autodesk.Revit.ApplicationServices;
+using Autodesk.Revit.Attributes;
+using Autodesk.Revit.DB;
+using Autodesk.Revit.UI;
+#endregion
+
+namespace HOK.ColorBasedIssueFinder
+{
+  class App : IExternalApplication
```

Comment:
I think we can use RevitToolkit's implementation of `ExternalApplication` here: https://github.com/Nice3point/RevitToolkit?tab=readme-ov-file#externalapplication